lib-injection/auto_inject.rb

```diff
@@ -0,0 +1,84 @@
+return if ENV['skip_autoinject']
+
+begin
+  require 'open3'
+
+  _, status = Open3.capture2e({'skip_autoinject' => 'true'}, 'bundle show ddtrace')
+
+  if status.success?
+    puts '[DATADOG LIB INJECTION] ddtrace already installed... skipping auto-injection'
+    return
+  end
+
+  require 'bundler'
+  require 'shellwords'
+
+  if Bundler.frozen_bundle?
+    puts "[DATADOG LIB INJECTION] Cannot inject with frozen Bundler"
```

Comment:
As documented in https://github.com/DataDog/dd-trace-rb/blob/master/docs/GettingStarted.md#custom-logging , we usually use `[ddtrace]` (or `[DDTRACE]` in a few cases), when logging this kind of information.

Unless `[DATADOG LIB INJECTION]` is a kind of "standard" of all lib injection scripts, I would suggest still using `[ddtrace]` here, so it's easy for customers to locate all ddtrace-related logs (e.g. when support asks for them), as otherwise they may miss these.